exec/Main.hs

```diff
@@ -7,7 +7,7 @@ module Main where
 
 import Universum
 
-import qualified Data.ByteString as BS
+import Data.ByteString qualified as BS
```

Comment:
I can see that you make such a changes in your first commit and at the second one
Please, make smth with it - move this changes from first commit to this one or maybe it'll be ok to squash this two commits (but in this case you need update commit title - and provide commit body! with description of your changes)